### ISSUE TITLE
Option to use Vanilla Shaderpack and Resourcepack Folders

### ISSUE THREAD

**Issue by Gregor7008** [modrinth/theseus#1064]
### Is your suggested enhancement related to a problem? Please describe.

I've played on the vanilla launcher for quite some time now and therefore built up quite the library of shaderpacks, mods, datapacks and resourcepacks. As I'd like to keep them when playing on Modrinth, I would have to either move all of them, copy all of them or link them using something like mklink. Doing that for every profile though is quite some work.

### Describe the solution you'd like

A simple switch somewhere in the settings that makes those links automatically would really help with this issue, maybe next to an upcoming "Only use one profile" switch :wink:.

### Describe alternatives you've considered

Without something like that, it is quite repelling for me and other players, who have used the default launcher for a decade now, to switch to Modrinth permanently.

**Comment by Prospector** [comment]
I don't think we plan to add this, but we do have plans to address global content
